cms/djangoapps/contentstore/views/item.py

```diff
@@ -574,7 +574,9 @@ def _duplicate_item(parent_usage_key, duplicate_source_usage_key, user, display_
         duplicate_metadata = {}
         for field in source_item.fields.values():
             if field.scope == Scope.settings and field.is_set_on(source_item):
-                duplicate_metadata[field.name] = field.read_from(source_item)
+                # Do not duplicate 'discussion_id' TNL-1754
+                if field.name != 'discussion_id':
```

Comment:
I don't see why `discussion_id` has to be changed when a discussion is copied. The `discussion_id` specifies a specific thread in the comments service backend, and it seems perfectly reasonable to me that multiple discussion Blocks might want to display the same thread (and thus, have the same `discussion_id`). Can you explain the rational for why having two discussion blocks with the same `discussion_id` is problematic?